src/plugins/credential_management/public/components/create_credential_wizard/components/header/header.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+
+import { EuiBetaBadge, EuiSpacer, EuiTitle, EuiText, EuiCode, EuiLink } from '@elastic/eui';
+
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import { DocLinksStart } from 'opensearch-dashboards/public';
+import { useOpenSearchDashboards } from '../../../../../../opensearch_dashboards_react/public';
+import { CredentialManagementContext } from '../../../../types';
+export const Header = ({
+  isBeta = true,
+  docLinks,
+}: {
+  isBeta?: boolean;
+  docLinks: DocLinksStart;
+}) => {
+  const changeTitle = useOpenSearchDashboards<CredentialManagementContext>().services.chrome
+    .docTitle.change;
+  const createCredentialHeader = i18n.translate('credentialManagement.createIndexPatternHeader', {
+    defaultMessage: 'Save Your Credential',
+  });
+
+  changeTitle(createCredentialHeader);
+
+  return (
+    <div>
+      <EuiTitle>
+        <h1>
+          {createCredentialHeader}
+          {isBeta ? (
+            <>
+              {' '}
```

Comment:
why need space here

Reply:
It's for consistent with index pattern management page. src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/header/header.tsx